locale/src/locale.cc

```diff
@@ -51,7 +51,11 @@ namespace leatherman { namespace locale {
 
     string translate(string const& s, string const& domain)
     {
-        return boost::locale::translate(s).str(get_locale("", domain));
+        try {
+            return boost::locale::translate(s).str(get_locale("", domain));
+        } catch (exception const&) {
+            return s;
```

Comment:
My reasoning here is:

* If locales are broken we should still return *something* from this function, instead of panicking
* Locale init needs to be done before we can log, so either a) we can't log, or b) the application has initialized logging, and had an opportunity to report to the user that localization is busted